data/json/mapgen/nested/house_nested.json

```diff
@@ -1546,5 +1546,633 @@
+      },
+      "place_loot": [
+        { "group": "SUS_hair_drawer", "x": 2, "y": 2, "chance": 50, "repeat": [ 1, 2 ] },
+        { "chance": 100, "item": "corpse_generic_male", "x": 2, "y": 2 },
+        { "chance": 100, "group": "male_underwear_bottom", "x": 2, "y": 2 },
+        { "chance": 50, "group": "pants_male", "x": 2, "y": 2 },
+        { "chance": 50, "group": "clothing_work_torso", "x": 2, "y": 2 },
+        { "chance": 50, "group": "shoestore_shoes", "x": 2, "y": 2 }
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_2x3_NE4",
+    "object": {
+      "mapgensize": [ 3, 3 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "t, ",
+        "j, ",
+        "{~ "
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "liquids": { "T": { "liquid": "water", "amount": [ 0, 50 ] } },
+      "items": {
+        "t": { "item": "SUS_toilet", "chance": 80 },
+        "j": [ { "item": "SUS_bathroom_sink", "chance": 50, "repeat": [ 1, 2 ] }, { "item": "SUS_bathroom_medicine", "chance": 50, "repeat": [ 1, 2 ] }, ],
+        "~": { "item": "SUS_bathroom_cabinet", "chance": 60, "repeat": [ 1, 2 ] }
+      },
+      "place_monsters": [ { "monster": "GROUP_BLOB", "chance": 50, "x": 1, "y": 1 } ],
+      "place_loot": [ { "group": "SUS_hair_drawer", "x": 2, "y": 2, "chance": 50, "repeat": [ 1, 2 ] } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_2x1_SW",
+    "object": {
+      "mapgensize": [ 2, 2 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "tj",
+        "  "
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "items": {
+        "t": { "item": "SUS_toilet", "chance": 80 },
+        "j": [ { "item": "SUS_bathroom_sink", "chance": 50, "repeat": [ 1, 2 ] }, { "item": "SUS_bathroom_medicine", "chance": 50, "repeat": [ 1, 2 ] }, ],
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_2x1_SW2",
+    "object": {
+      "mapgensize": [ 2, 2 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "tj",
+        "  "
+      ],
+      "palettes": [ "house_w_nest_palette" ],
+      "items": {
+        "t": { "item": "SUS_toilet", "chance": 80 },
+        "j": [ { "item": "SUS_bathroom_sink", "chance": 50, "repeat": [ 1, 2 ] }, { "item": "SUS_bathroom_medicine", "chance": 50, "repeat": [ 1, 2 ] }, ],
+      },
+      "place_monster": [ { "monster": "mon_sludge_crawler", "chance": 25, "x": 1, "y": 1 } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates location of door",
+    "nested_mapgen_id": "bathroom_2x1_SW3",
```

Comment:
I don't think these should all be different nested_mapgen_ids, they have the same floorplan and just different chances of something interesting showing up. If you give them all the same ID, the game will randomly select one bathroom_2x1_SW from all possible IDs, which is generally more desirable behaviour.